docs/fe-testlib.md

```diff
@@ -0,0 +1,178 @@
+# 📚 Test Library Integration
+
+A test library provides test functions that can be used to safeguard your Fiori / UI5 app in combination with your own test functions.
+
+## Why should you use the SAP Fiori elements for OData V4 test library?
+
+The SAP Fiori elements test library provides a set of test functions that can be reused within [OPA5](https://sapui5.hana.ondemand.com/#/topic/2696ab50faad458f9b4027ec2f9b884d) or [wdi5](https://github.com/js-soft/wdi5) tests of [SAP Fiori elements](https://experience.sap.com/fiori-design-web/smart-templates/) for OData V4 applications. When you write such tests, you only need to test the application-specific functionalities, e.g. a custom action. The core functionalities provided by SAP Fiori elements are already tested by the framework itself.
+
+Nevertheless, you need to implement certain steps to test the specific functionalities of the app, e.g. loading data into the list-report, navigating between a list-report and an object-page, saving business objects, etc. To implement these steps within a WDI5 test, you can reuse the test functions of the OData V4 OPA test library and you don´t need to re-implement these functions. This saves time and avoids redundancy.
+
+Another advantage of using the OData V4 test library is that updates for the test functions due to framework changes are automatically provided by the framework. This ensures that your tests will not break.
+
+### Main benefits of using the test library
+
+- Reduces test maintenance efforts and avoid code repetition.
+- Isolate generic actions and validations in the test library and reuse them across apps.
+- Tests are simplified and have compact page objects and short journeys.
```

Comment:
Journeys are not necessarily short when using the test lib

Reply:
Removed as we also have no journeys in wdi5